### ISSUE TITLE
Suggestion: Use last selected dino in library as starting point in Extractor

### ISSUE THREAD

**Issue by Jagged** [cadon/ARKStatsExtractor#21]
I'm breeding several types of dinos and often the babies differ just slightly.  Presently I'm having to rekey everything as the Extractor defaults are typically unhelpful.

Edit:  I see I can now achieve that thru a two step process: (1) Right click the dino to open the Stat Testing screen, and then (2) Click on the "Copy Stat-Values to Extractor" button.

It's a step in the right direction at least!  But any thoughts re my initial comment?

**Comment by cadon** [comment]
Hi, can you rephrase your request, what do you mean by "rekey"? Do you mean the choosen combination after the extraction is not the one you expect?
If you add creatures to the lib, the numbers you typed in are preserved, so you only have to change the ones that are different.

**Comment by Jagged** [comment]
> Do you mean the chosen combination after the extraction is not the one you expect?
No, I'm referring to having to enter the individual stats of the new dino before I hit the extract button.

>If you add creatures to the lib, the numbers you typed in are preserved, so you only have to change the ones that are different.
That's only useful if the new dino is the same type as the last one I used extract for.

If the extract screen could pull in the stats of the dino last selected in the library we would only then have to enter the differences.  It would essentially allow parents or similar dinos to act as a template and save typing when loading new dinos.

Cheers, Will =)

**Comment by cadon** [comment]
Ah, now I get it. I'm planning on doing a simpler adding process for offspring where you just have to choose between the two possible stats of the parents. I'm currently waiting for the Breeding Phase 2 to be released to see how that would affect this feature.

**Comment by cadon** [comment]
You can now copy the values of a creature directly to the Extractor by right-clicking -> Copy Values to Extractor.
